opengeo/settings.py

```diff
@@ -110,6 +110,10 @@ class TestingConfiguration(OpenGeoMixin, TestingBaseConfiguration):
     CELERY_TASK_EAGER_PROPAGATES = True
 
 
+class NonDebugDevConfiguration(DevelopmentConfiguration):
+    DEBUG = True
```

Comment:
@zachmullen, this is copied directly from your work in https://github.com/ResonantGeoData/ResonantGeoData/pull/334/files#diff-2c77f7552f3fa334decbc44137040144e567d15cbb3c076a6dd71f0ffbe8ba79R99-R100

This name is tripping me up... "NonDebug" implies `DEBUG = False` to me. Would you clarify this, please?

Reply:
Oops, I think that's just a typo... it definitely should be `DEBUG = False`, good catch.